addons/account/views/partner_view.xml

```diff
@@ -225,6 +226,17 @@
                                 <field name="property_account_receivable_id"/>
                                 <field name="property_account_payable_id"/>
                             </group>
+                            <group string="Credit Limits"
+                                   name="credit_limits"
+                                   groups="account.group_account_invoice,account.group_account_readonly"
+                                   attrs="{'invisible': [('show_credit_limit', '=', False)]}">
+                                <field name="credit"/>
```

Comment:
From the discussion we had last night with TSB, we can keep it as is for now, since there are other similar fields that are already being computed on the smart buttons, right ?